src/gssapi/dse_auth_gssapi.cpp

```diff
@@ -22,7 +22,6 @@
 #include <string.h>
 
 #include <gssapi/gssapi.h>
-#include <gssapi/gssapi_generic.h>
```

Comment:
This header file isn't included in the krb5 vcpkg build.  At least for modern versions of the gssapi lib this header isn't [especially interesting](https://github.com/krb5/krb5/blob/master/src/lib/gssapi/generic/gssapi_generic.h) so it seemed easy enough to remove.